Sources/UIKit/AdaptiveStyle.swift

```diff
@@ -10,37 +10,74 @@
 import UIKit
 
 /// A few default font scaling behaviors.
-public enum AdaptiveStyle {
+public struct AdaptiveStyle {
+
+    public enum Behavior {
+        case control
+        case body
+        case preferred
+
+        @available(iOS 11, tvOS 11, *)
```

Comment:
I checked, and apparently the stuff we use for this actually goes back to iOS 10, but I don't see any point in back-porting it now.